frontend/src/pages/PickupGem/FoundGem.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <div class="bg-container">
+    <video autoplay muted :poster="getPoster" id="gem-video">
+      <source :src="getVideo" type="video/mp4">
+    </video>
+    <div id="found-gem">
+      <h1>You found a gem! {{ gemPackage.color }}</h1>
```

Comment:
Is it intentional that the color is mentioned here? Feels quite out of place haha

Reply:
omg thanks for pointing it out LOL I think I was testing and then forgot to delete